app/models.js

```diff
@@ -63,7 +63,8 @@ export type Event = EventBase & {
   company: Company,
   comments: Array<Comment>,
   pools: Array<EventPool>,
-  survey: ?ID
+  survey: ?ID,
+  userReg: Object
```

Comment:
You can use `userReg: EventRegistration` here instead. (see line `100` in this file).